packages/react-app/src/App.tsx

```
@@ -83,20 +83,19 @@ function App() {
         />;
       case 'analyticsPage':
         return <AnalyticsPage />;
+      case 'accountsPage':
+        return <AccountsPage />;
       case 'dashboard':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)